at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb

```diff
@@ -0,0 +1,38 @@
+class StringFormat
+  def self.human_name
+    out = ''
+    _a = 0
+    p 'Enter input:'
+    str = gets.chomp
+    str.strip!
+    x = str.gsub(/[^A-Za-z ]/, '')
+    for i in 0...x.length do
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/2f1c4a77b40d8a5108a73d18074d99a3676e22fe/at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb#L9"></span>Style/For: Prefer <code>each</code> over <code>for</code>. (https://github.com/rubocop-hq/ruby-style-guide#no-for-loops)</td>
    </tr>
  </tbody>
</table>